allure-junit-platform/src/main/java/io/qameta/allure/junit5/AllureJunit5.java

```diff
@@ -72,15 +74,43 @@ public void executionStarted(final TestIdentifier testIdentifier) {
 
             methodSource.ifPresent(source -> {
                 result.setDescription(getDescription(source));
-                result.getLabels().add(new Label().withName("suite").withValue(getSuite(source)));
-                result.getLabels().add(new Label().withName("package").withValue(source.getClassName()));
+                result.getLabels().add(new Label().withName(SUITE).withValue(getSuite(source)));
+                result.getLabels().add(new Label().withName(PACKAGE).withValue(source.getClassName()));
             });
 
             getLifecycle().scheduleTestCase(result);
             getLifecycle().startTestCase(uuid);
         }
     }
 
+    @Override
+    public void executionSkipped(TestIdentifier testIdentifier, String reason) {
```

Comment:
quality plugin will fail on missing `final` for inputs

Reply:
I'll look at checks, thanks